event/autonat.go

```diff
@@ -0,0 +1,5 @@
+package event
```

Comment:
Could we change this filename to something more abstract that doesn't reference a particular subsystem? `routability.go`?